README.md

```diff
@@ -21,8 +21,8 @@ The file licenses.md is a table of contents for the generated licenses.  The lin
 Technical questions or issues can be sent to the [SPDX technical team mailing list](mailto:spdx-tech@lists.spdx.org).
 Questions or issues on the license content can be sent to the [SPDX legal team mailing list](mailto:spdx-legal@lists.spdx.org).
 
-Note that the content of this repository is generated by the [SPDX tools](http://github.com/spdx/tools) from the [license list source](http://github.com/spdx/license-list).
+Note that the content of this repository is generated by the [SPDX tools](http://github.com/spdx/tools) from the [license list source](https://github.com/spdx/license-list-XML).
 
-As far as the *content* of the license information is concerned, any [issues](https://github.com/spdx/license-list/issues) or [pull requests](https://github.com/spdx/license-list/pulls) should be logged at the [license list source](http://github.com/spdx/license-list) project.
+As far as the *content* of the license information is concerned, any [issues](https://github.com/spdx/license-list-XML/issues) or [pull requests](https://github.com/spdx/license-list-XML/pulls) should be logged at the [license list source](https://github.com/spdx/license-list-XML) project.
```

Comment:
I don't expect repository changes/renames to be frequent, but we can make this a bit DRYer as well as making the source slightly easier to read by using reference-style links:

```markdown
As far as the *content* of the license information is concerned, any [issues][canonical-list-issues] or [pull requests][canonical-list-pulls] should be logged at the [license list source project][canonical-list].

[canonical-list]: https://github.com/spdx/license-list-XML
[canonical-list-issues]: https://github.com/spdx/license-list-XML/issues
[canonical-list-pulls]: https://github.com/spdx/license-list-XML/pulls
```